net/basicstation/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=basicstation
+PKG_VERSION:=2.0.6
+PKG_RELEASE:=1
```

Comment:
As this is inspired from https://github.com/xueliu/lora-feed/commits/master/basicstation, why don't you use autorelease?

Reply:
I missed that, but now it is updated.